fixtures/stress_test/Sources/FixtureGenerator/Templates/ManifestTemplate.swift

```diff
@@ -0,0 +1,70 @@
+
+import Foundation
+
+class ManifestTemplate {
+    private let workspaceTemplate = """
```

Comment:
Would it make sense to use [swift-format](https://github.com/apple/swift-format) to generate the manifests? I think it's easier to maintain that using strings.

Reply:
Indeed, managing those strings can easily get messy. It will do for those simple cases at the moment but perhaps requires something more advanced later on.

Did you mean [`SwiftSyntax` ](https://github.com/apple/swift-syntax)? I believe that library allows generating swift code.

Reply:
> Did you mean SwiftSyntax ? I believe that library allows generating swift code.

Sorry yes! That's what I meant. Something to keep in mind for future iterations of this code. As you suggested, if dealing with strings gets messy, we can change the approach.